compiler/testData/codegen/boxAgainstJava/sam/adapters/localClass.kt

```diff
@@ -0,0 +1,8 @@
+var status: String = "fail"
```

Comment:
I beleive it would be good to add comment explaining why "status" here is top-level property instead of local variable.